processing file asmFiles/halt.asm (1 of 1)
processing file asmFiles/halt.asm (1 of 1)
--- memsim.hex	2021-10-25 13:10:47.677084010 -0400
+++ memcpu.hex	2021-10-25 13:10:47.604081367 -0400
@@ -7,4 +7,5 @@
 :04000600AC41000801
 :04000700AC44000CF9
 :04000800AC410010F7
+:040020000000000AD2
 :00000001FF
